canonical-web-and-design/jaas-dashboard
# Filter entities by application on model details

## Done

When clicking an application on the model details page it'll now filter down the units, machines, and relations that are that application.

## QA

- Pull code
- Run `./run clean && ./run serve`
- Open http://0.0.0.0:8036/
- View a model details and click one of the applications. It should filter the units, machines, and relations to only show ones for that application.
- View a model details with subordinates, be sure that it also highlights subordinates properly.
- View a model details with subordinates that aren't on each parent unit, be sure that it only shows units with subordinates.

Contact me if you don't have models that match these criteria.

## Details

Fixes #148 

## Screenshots

<img width="1632" alt="Screen Shot 2020-02-05 at 2 29 55 PM" src="https://user-images.githubusercontent.com/532033/73880415-0483f200-4824-11ea-83ee-dc804085f4cd.png">
